src/main/java/racingcar/view/ConsoleView.java

```diff
@@ -0,0 +1,44 @@
+package racingcar.view;
+
+import camp.nextstep.edu.missionutils.Console;
+import java.util.List;
+import racingcar.model.Car;
+
+public class ConsoleView {
+    public static final String START_INPUT_MESSAGE = "경주할 자동차 이름을 입력하세요. (이름은 쉼표(,)로 구분)";
+    public static final String TRY_COUNT_INPUT_MESSAGE = "시도할 횟수는 몇 회인가요?";
+    public static final String RESULT_OUTPUT_MESSAGE = "실행 결과";
+    public static final String WINNER_OUTPUT_MESSAGE = "최종 우승자 : ";
+
+    public void printStartMessage() {
+        System.out.println(START_INPUT_MESSAGE);
+    }
```

Comment:
```suggestion
    public String printStartMessage() {
        System.out.println(START_INPUT_MESSAGE);
        readInput();
    }
```
와 같이 해주면 어떨까요? 개인적으로 외부에서 두 개의 메서드를 따로 호출할 필요성이 없다고 느껴지네요!
어차피 입력하기 전에는 항상 메시지를 출력을 해줄거니까요ㅎㅎ 😁 
두 개의 메서들 각각 호출해주신 이유가 궁금합니다!

Reply:
입력하기 전 항상 메시지를 출력한다는 말엔 공감합니다 

두 개의 메서드를 각각 호출한 이유는, 출력을 하는 print() 메서드와 입력을 받는 input() 메서드의 역할은 다르다고 생각했기 때문입니다. 그렇기에 출력을 하는 메서드에서 입력을 받는 것은 맞지 않다고 생각했습니다. 

외부에서 메서드를 많이 호출하면 안 좋은 점이 있은 걸까요? 너무 작게 메서드를 나눈 건지 궁금합니다!